src/components/Courses/Courses.jsx

```diff
@@ -4,10 +4,9 @@ import CourseCard from '../CourseCard/CourseCard';
 import { Button } from '../Button/Button';
 import InputField from '../Input/Input';
 import { Link } from 'react-router-dom';
-import { fetchDataGo } from '../../utils/api';
-import { actionCreators } from '../../store/courses/actionCreators';
-import { actionCreators as actionCreatorsAuthors } from '../../store/authors/actionCreators';
 import { useSelector, useDispatch } from 'react-redux';
+import { thunk_getCourses } from '../../store/courses/thunk';
```

Comment:
no need to explicitly mark with `thunk_` prefix your thunk-based action creators. If you think this is really needed, I'd recommend use camelCase for all your action creator namings